src/setup-tflint.js

```diff
@@ -102,6 +124,7 @@ async function installWrapper(pathToCLI) {
 async function run() {
   try {
     const inputVersion = core.getInput('tflint_version');
+    const expectedHashes = core.getInput('expected_sha256')?.split(',').map(hash => hash.trim()) || undefined;
```

Comment:
Splitting on newlines as well is a nice benefit as actions commonly do both but newline-delimited is better for readability for anything long.